### DISCUSSION TITLE
Prune Non-Existent Paths in Database

### DISCUSSION THREAD

**Discussion by razmatei** [ajeetdsouza/zoxide#557] [Q&A] [answered]
Hello. Is there a way to automatically prune non-existent/removed paths from the zoxide DB ? Thank you

**Comment by ajeetdsouza** [comment]
Is there any reason you want to do this manually? zoxide auto-prunes non-existent entries that have not been accessed in the last 90 days.

**Comment by razmatei** [comment]
No reason to do it manually no. I just probably missed the part in the docs where the 90 days auto-pruning is mentioned.

**Reply by ajeetdsouza** [reply, marked as answer]
I hadn't mentioned it because I felt that users shouldn't have to think of implementation details as long as the command runs fast. However, I've added a small section [here](https://github.com/ajeetdsouza/zoxide/wiki/Algorithm#pruning).
